### PR TITLE
UI Setup

### PR DESCRIPTION
**Pull request by flore77**
For details: #10

### CODE REVIEW

**Comment by flore77** [comment]
@andrei-picus-hs please review here. Sorry but I had to close the other pull-request, because the target branch was the master...

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  entry: './ui/entry.jsx',
+  output: {
+    filename: './statics/bundle.js'
+  },
+  module: {
+    loaders: [
+      {test: /\.js$/, loader: 'babel-loader'},
+      {test: /\.jsx$/, loader: 'babel-loader'},
```
Comment (andrei-picus-hs) [inline webpack.config.js]: You can unify the two regexps into `/\.jsx?$/`. `x?` means 0 or 1 occurrences.

File: statics/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'/>
```
Comment (andrei-picus-hs) [inline statics/index.html]: Use double quotes.

File: statics/index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    <title>Mughsot Visual Report</title>
+    <script src='../data.js'></script>
```
Comment (andrei-picus-hs) [inline statics/index.html]: Add `type="text/javascript"`.

File: ui/entry.jsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+var React = require('react'),
+    Report = require('./components/report.jsx');
+
+require('./styles/base.css');
+
+React.render(<Report data={data}/>, document.getElementById('report'));
```
Comment (andrei-picus-hs) [inline ui/entry.jsx]: 1. You need to tell ESLint to lint `.jsx` files as well. You do this by calling `eslint --ext js --ext jsx`.
2. ESLint will tell you that `data` is undefined. Tell it it's a global using `/* globals data */`.
3. You can get the name of the variable by using

```js
var reportGenerator = require('lib/report-generator.js'),
    data = window[reportGenerator.MOCHA_TEST_DATA_VAR_NAME];
```
4. You might not need the globals pragma anymore if you access the variable from `window`.

**Comment by andrei-picus-hs** [comment]
Finished review. @flore77 ping me if you run into trouble with ESLint.

**Comment by flore77** [comment]
@andrei-picus-hs please review updates :smiley:

**Comment by andrei-picus-hs** [comment]
LGTM

**Pull request merged**